cmd/tailscale/cli/configure-kube.go

```diff
@@ -44,6 +44,10 @@ See: https://tailscale.com/s/k8s-auth-proxy
 
 // kubeconfigPath returns the path to the kubeconfig file for the current user.
 func kubeconfigPath() string {
+	if kubeconfig := os.Getenv("KUBECONFIG"); kubeconfig != "" {
+		return filepath.SplitList(kubeconfig)[0]
+	}
+
 	var dir string
 	if version.IsSandboxedMacOS() {
```

Comment:
I just noticed this line. Setting KUBECONFIG while also running the sandboxed (App Store variant) build would result in weirdness. Probably better to just return an error if kubeconfig != "" and they're using the sandboxed build.

Reply:
To return an error I had to convert the function, but it's only used in one place anyway.
An alternative could be to panic.

I'm adding it as a separate commit so it can be reverted if it turns out not to be a problem in practice (since it modifies a little bit more than previously).